mobile/src/screens/ChatScreen.tsx

```diff
@@ -1,21 +1,31 @@
 import React from 'react';
 import { Text, View, StyleSheet } from 'react-native';
+import { TextStyles } from '../styles/TextStyles';
 
 /**
  * Left tab on navbar for chatting with volunteers
  */
 export const ChatScreen: React.FC = () => {
   return (
     <View style={styles.container}>
-      <Text>Chat</Text>
+      <Text style={TextStyles.h3}>  {`\tLet's Talk\n`}</Text>
+      <Text> <Text style={TextStyles.h3}> {`\t`}Tel: </Text><Text>858.274.9673{`\n`}</Text> </Text>
+      <Text> <Text style={TextStyles.h3}> {`\t`}Fax: </Text><Text>858.274.9673{`\n`}</Text> </Text>
+      <Text> <Text style={TextStyles.h3}> {`\t`}Mon-Fri: </Text><Text>8am-4pm{`\n`}</Text> </Text>
+      <Text> <Text style={TextStyles.h3}> {`\t`}Email Us: </Text><Text>info@wordsalive.gov{`\n`}</Text> </Text>
+      <Text> <Text style={TextStyles.h3}> {`\t`}Address: </Text><Text>{`5111 Santa Fe Street Suite\n\t219 San Diego, California, 92081, \n\tUnited States`}{`\n`}</Text> </Text>
```

Comment:
Why are these double-wrapped in `<Text>` components?

Reply:
Also as discussed, you should use CSS for styling, not tabs and line breaks.

Reply:
Lastly, this is only a temporary screen but we should still make all of the text internationalized, can you add these strings to i18n/?

Reply:
Also you're missing text styles on some of these text tags (the parts after the colon) so the font doesn't match. Can you define a new style with the same size but with the light font variant?